docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.post_update.php

```diff
@@ -183,3 +184,18 @@ function su_humsci_profile_post_update_8202() {
       ->save();
   }
 }
+
+/**
+ * Copy the media module icon to the site's files directory.
```

Comment:
Why?

Reply:
these icons are used in the media library as the "default images" even for documents and pdfs.